config/routes.rb

```diff
@@ -24,6 +24,7 @@
       end
       resources :moves, only: %i[index show create destroy update] do
         resources :documents, only: %i[create destroy]
+        resources :events, only: %i[create], controller: 'move_events'
```

Comment:
Are we using `controller: move_events` to avoid nesting the controller in a `Move` module?

Reply:
hmm... good question; don't mind either way - which is more railsy ?

Reply:
I think not overriding the controller (principle of convention over configuration) makes more sense.

Reply:
But we're also doing it loads elsewhere in this file

Reply:
so I've experimented moving `MoveEventsController` to `EventsController` in the `Moves` module... however, this breaks a lot of other stuff reliant on the `Moves` module/namespace (`Moves::ParamsValidator`, `Moves::Finder`, Moves::ImportPeople` etc). 

I guess there is something wrong with the implementation of the moves namespace; but I think I'll leave that as a separate issue to resolve later and keep with MoveEventsController for now.